test/src/validators/validators_test.dart

```diff
@@ -773,11 +773,33 @@ void main() {
     test('Compare a list of numbers with another that contains a part of them (invalid)', () {
       final control = FormControl<List<int>>(
           value: [1,2,3,4],
-          validators: [Validators.contain([1,3,5])],
+          validators: [Validators.contains([1,3,5])],
       );
 
       // Expect: control invalid
       expect(control.valid, false);
     });
+
+    test('Compare a list of emails with another that contains numbers (invalid)', () {
+      final control = FormArray<String>([
+        FormControl<String>(value: 'john@email.com'),
+        FormControl<String>(value: 'stark@email.com'),
+        FormControl<String>(value: 'bob@email.com'),
+      ], validators: [Validators.contains([1,3,5])]);
```

Comment:
This test is more readable if instead of validating: Validators.contains ([1,3,5]) you use Validators.contains (['doe@email.com'])